[*] Running ...
[*] Dataset loaded: C:\Users\Nam\Documents\GitHub\HSEvo/problems/tsp_aco\dataset\train50_dataset.npy with 5 instances.
C:\Users\Nam\Documents\GitHub\HSEvo\problems\tsp_aco\gpt.py:21: RuntimeWarning: invalid value encountered in multiply
  u = np.argmin(dist + visited * np.inf)
[*] Instance 0: 6.04662144143095
[*] Instance 1: 5.977115757779524
[*] Instance 2: 7.0460923676543565
[*] Instance 3: 6.608129594928318
[*] Instance 4: 6.284783748284308
[*] Average:
6.3925485820154915
